### PR TITLE
Remove Scraper

### PR DESCRIPTION
**Pull request by Ian-Stewart-Binks**
Signed-off-by: Ian Stewart-Binks istewart@redhat.com

### CODE REVIEW

**Comment by Anirudh94** [comment]
do you also want to remove the .travis.yml and Gruntfile.js flake8 related stuff?

**Comment by Ian-Stewart-Binks** [comment]
@Anirudh94 Yeah. I'll do that.

File: Gruntfile.js [hunk 1]
```diff
@@ -9,7 +9,6 @@ module.exports = function(grunt) {
     });
 
     grunt.loadNpmTasks('grunt-eslint');
-    grunt.loadNpmTasks('grunt-flake8');    
     grunt.registerTask('lint', 'eslint');
     grunt.registerTask('default', ['lint', 'flake8']);
```
Comment (Anirudh94) [inline Gruntfile.js]: Oops forgot to remove the 'flake8' task ;)

**Comment by Ian-Stewart-Binks** [comment]
@Anirudh94 I'm going to merge this in now.

**Pull request merged**